lib/evaluation/evaluationClient.ts

```diff
@@ -41,11 +42,29 @@ export class EvaluationClient {
     return this._results;
   }
 
-  get results(): HandlerCheckResult[] {
+  public evaluateWrapper(pd: PresentationDefinition, vp: unknown): EvaluationResults {
```

Comment:
the method described in the spec is called `evaluate` and I think it's better to actually have another file for our real world methods such as `selectFrom`, `evaluate`, and `submissionFrom`

Reply:
please see the content of this file: https://github.com/Sphereon-Opensource/pe-js/blob/feature/ESSIFI-108/lib/evaluation/evaluationClientWrapper.ts

Reply:
Agreed.